spring-caching/src/test/java/com/baeldung/ehcache/SquareCalculatorUnitTest.java

```diff
@@ -20,21 +25,24 @@ public void setup() {
     @Test
     public void whenCalculatingSquareValueOnce_thenCacheDontHaveValues() {
         for (int i = 10; i < 15; i++) {
-            assertFalse(cacheHelper.getSquareNumberCache().containsKey(i));
-            System.out.println("Square value of " + i + " is: " + squaredCalculator.getSquareValueOfNumber(i) + "\n");
+            assertFalse(cacheHelper.getSquareNumberCache()
+                    .containsKey(i));
+            LOGGER.debug("Square value of {} is: {}", i, squaredCalculator.getSquareValueOfNumber(i));
         }
     }
 
     @Test
     public void whenCalculatingSquareValueAgain_thenCacheHasAllValues() {
         for (int i = 10; i < 15; i++) {
-            assertFalse(cacheHelper.getSquareNumberCache().containsKey(i));
-            System.out.println("Square value of " + i + " is: " + squaredCalculator.getSquareValueOfNumber(i) + "\n");
+            assertFalse(cacheHelper.getSquareNumberCache()
```

Comment:
Keep the original formatting.

Reply:
Yep done. I need to update the code style in my IDE. will take care of that next time :)